game-engine/src/test/scala/Winning_State_Tests.scala

```diff
@@ -0,0 +1,108 @@
+import org.scalatest.FunSuite
+import test.TestHelper
+import za.co.entelect.challenge.game.contracts.Config.Config
+import za.co.entelect.challenge.game.contracts.command.RawCommand
+import za.co.entelect.challenge.game.contracts.commands.CommandFactory
+import za.co.entelect.challenge.game.contracts.game.CarGamePlayer
+import za.co.entelect.challenge.game.contracts.map.CarGameMap
+import za.co.entelect.challenge.game.contracts.renderer.CarGameRendererFactory
+
+class Winning_State_Tests extends FunSuite {
+
+  private val nothingCommandText = "NOTHING"
+
+  private var commandFactory: CommandFactory = null
+  private var nothingCommand: RawCommand = null
+
+  def initialise() = {
+    Config.loadDefault()
+    commandFactory = new CommandFactory
+
+    nothingCommand = commandFactory.makeCommand(nothingCommandText)
+    nothingCommand.setCommand(nothingCommandText)
+  }
+
+  test("Given the match ends due to max rounds reached player 1 needs to win because of being furthest on the track") {
```

Comment:
Shouldn't this be phrased as `Given>When>Returns` instead?
`Given player 1 is ahead, When max rounds limit is reached, Player 1 should win`
We don't have a wiki for these, so no change required as long as it's consistent

Does scala have a test grouping/nesting mechanism? This way you could wrap all these cases into a "Given the match ends due to max rounds reached" group to structure them around common logic